README.md

```diff
@@ -15,11 +15,10 @@ Inspired by Gnome Shell Top Bar indicators.
 
 ### Extensions
 
-- [jupyterlab-topbar-extension](./packages/jupyterlab-topbar-extension): generic extension to expose the top bar area
-- [jupyterlab-topbar-text](https://github.com/jupyterlab-contrib/jupyterlab-topbar-text): add and edit custom text
-- [jupyterlab-system-monitor](https://github.com/jtpio/jupyterlab-system-monitor): show system metrics (memory usage)
-- [jupyterlab-logout](https://github.com/jupyterlab-contrib/jupyterlab-logout): add a "Log Out" button
-- [jupyterlab-theme-toggle](https://github.com/jtpio/jupyterlab-theme-toggle): switch between the Light and Dark themes
+- [jupyterlab-topbar-text](./packages/topbar-text-extension): add and edit custom text
+- [jupyterlab-system-monitor](./packages/system-monitor-extension): show system metrics (memory usage)
```

Comment:
For `jupyterlab-system-monitor` either we include it here, or maybe directly in `jupyter-resource-usage`? https://github.com/jupyter-server/jupyter-resource-usage/issues/191

Reply:
I agree that it makes more sense to move it over to `jupyter-resource-usage` as users can install everything at once. I will work on that!!

Reply:
that would be great, thanks!

Reply:
For reference there is a PR to update `jupyter-resource-usage` to JupyterLab 4: https://github.com/jupyter-server/jupyter-resource-usage/pull/195, that could be finished before https://github.com/jupyter-server/jupyter-resource-usage/issues/191.

Or update to JupyterLab 4 directly as part of https://github.com/jupyter-server/jupyter-resource-usage/issues/191.

Reply:
@jtpio How do we organise the `jupyter-resource-usage`? Should I wait for you to finish your [PR](https://github.com/jupyter-server/jupyter-resource-usage/pull/195) and then add the lab extension package?

Reply:
I'll have very low bandwidth over the next 2 weeks so if you want to take over and finish the lab 4 PR that would be greatly appreciated. Thanks!

Reply:
Sure, I can do that. Are you going to add me as a collaborator or should I start a new PR with your branch as base?